autoload/flog/state.vim

```diff
@@ -150,7 +150,8 @@ endfunction
 
 function! flog#state#GetCommitRefs(commit) abort
   let l:refs = []
-
+  let l:remotes = flog#shell#Run(flog#fugitive#GetGitCommand() . ' remote -v')
+  let l:remotes = uniq(map(l:remotes, 'substitute(v:val, "\t.*$", "", "")'))
   for l:ref in split(a:commit.refs, ', ')
     let l:match = matchlist(l:ref, '\v^(([^ ]+) -\> )?(tag: )?((refs/(remote|.*)?/)?((.{-}/)?(.*)))')
```

Comment:
I believe `(remote|.*)?/` might have to be changed to `(remote|.{-})?/` to make it non greedy to handle the case of `remote-name/branch/with/slashes`.